controller_interface/src/controller_interface_base.cpp

```diff
@@ -166,7 +167,8 @@ const rclcpp_lifecycle::State & ControllerInterfaceBase::configure()
     {
       RCLCPP_WARN(
         get_node()->get_logger(),
-        "The update rate of the controller : '%ld Hz' cannot be higher than the update rate of the "
+        "The update rate of the controller : '%" PRId64
+        "Hz' cannot be higher than the update rate of the "
```

Comment:
```suggestion
        "The update rate of the controller : '%zu"
        "Hz' cannot be higher than the update rate of the "
```

Heyy!

Will this work?

Reply:
hii.. let me check!

Reply:
@saikishor 

not working unfortunately!

```bash
/Users/dhruvpatel29/kilted-ros2/src/ros2_control/ros2_control/controller_interface/src/controller_interface_base.cpp:173:9: error: format specifies type 'size_t' (aka 'unsigned long') but the argument has type 'int64_t' (aka 'long long') [-Werror,-Wformat]
  170 |         "The update rate of the controller : '%zu"
      |                                               ~~~
      |                                               %lld
  171 |         "Hz' cannot be higher than the update rate of the "
  172 |         "controller manager : '%d Hz'. Setting it to the update rate of the controller manager.",
  173 |         update_rate, ctrl_itf_params_.update_rate);
      |         ^~~~~~~~~~~
      ```

Reply:
Can you test with %u please? Thank you

Reply:
The ultimate bikeshedding experience :D 
Anyone opposed to trying FMT? We already use it in other places...

Reply:
Much better hahaha. I agree with you